src/adv.py

```diff
@@ -38,14 +38,38 @@
 #
 
 # Make a new player object that is currently in the 'outside' room.
-
+player = Player('Irving', room['outside'])
+print(player)
+# userInput = ""
 # Write a loop that:
 #
+while True:
+
 # * Prints the current room name
+    print(player.location.name)
 # * Prints the current description (the textwrap module might be useful here).
+    print(player.location.description)
 # * Waits for user input and decides what to do.
-#
+    user_input = input(
+        "where do you want go? (N)orth, (S)outh, (E)ast, or (W)est?")
 # If the user enters a cardinal direction, attempt to move to the room there.
+    if(user_input == 'n' or user_input == 'N'):
+        target = player.location.n_to
+    # print(current_room.n_to)
+    elif(user_input == 's' or user_input == 'S'):
+        target = player.location.s_to
+    elif(user_input == 'e' or user_input == 'E'):
+        target = player.location.e_to
+    elif(user_input == 'w' or user_input == 'W'):
+        target = player.location.w_to
+    elif(user_input == 'q' or user_input == 'Q'):
+        break
+    else:
+        print("invalid input")
+
+    if(target != None):
+        player.move(target)
```

Comment:
nice OOP here, player's location is changed by a method belonging to the player.